docs/source-2.0/aws/aws-iam.rst

```diff
@@ -808,6 +738,87 @@ The computed condition keys for the service are:
           * ``aws:region``
 
 
+
+.. smithy-trait:: aws.iam#disableConditionKeyInference
+.. _aws.iam#disableConditionKeyInference-trait:
+
+``aws.iam#disableConditionKeyInference`` trait
```

Comment:
This trait needs to be before the "Deriving condition keys" section, as it is mentioned there.